# Underwater Shape Identification Challenge Parameters

template_directory         :  "templates/underwater_shape_identification/"
depth                      :  5.0
depth_uncertainty          :  1.0
shape_area                 :  1.0
min_detections             :  10
min_boat_displacement      :  0.1
is_target                  :  {"circle":true, "cross":true, "triangle":true}
detection_thresholds       :  {"circle":0.5, "cross":0.5, "triangle":0.5}

camera_topic               :  "/down/image_raw"
buffer_size                :  5

# calibration_images         :  "calib/underwater_shape_identification/" 
